### ISSUE TITLE
bug: ionTabsDidChange not firing

### ISSUE THREAD

**Issue by mccabemj** [ionic-team/ionic#19665]
# Bug Report

**Ionic version:**
[x] **4.11 React**

**Current behavior:**
`ionTabsDidChange` not working

**Expected behavior:**
`ionTabsDidChange` as per the documentation should fire when tab is changed

**Steps to reproduce:**
Base ionic tab app, adding

```
 <IonTabs
            ionTabsDidChange={() => { console.log('fired') }}
```

**Ionic info:**

```
Ionic:
   Ionic CLI       : 5.4.2 
   Ionic Framework : @ionic/react 4.11.1
```

**Comment by praveenptl71** [comment]
You can use like this

`<ion-tabs (ionTabsDidChange)="ionTabsDidChange($event)" >
</ion-tabs>`

**Comment by elylucas** [comment]
Hi @mccabemj ,

Thanks for the issue, this should be fixed in the upcoming 4.11.2 release which should be out today.
